DOCKER.md

```diff
@@ -0,0 +1,114 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# accumulo-proxy-docker
+
+A temporary guide on how to run this up in Docker.
```

Comment:
```suggestion
A temporary guide on how to run this in Docker.
```

Reply:
Nice spot! I completely missed that I had left it as a temporary guide when I first committed the file. I've now added a better description.

Reply:
This is looking really good.  I gave everything a quick once over and saw a few things.   Do you still consider this a WIP?

Reply:
I think I'm happy for it to ship, so any last minute things let me know and I'll aim to fix them today/tomorrow, otherwise I'm happy for it to merge whenever.

I removed WIP from the title of the pull request now.